src/app/shared/components/course-card/course-card.component.ts

```diff
@@ -1,8 +1,27 @@
-import { Component } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { mapAuthorIdsToNames } from '@app/utilities/author-utils';
+import { mockedAuthorsList } from '@shared/mocks/mocks';
+import { Course } from '@features/courses/course';
 
 @Component({
   selector: 'app-course-card',
   templateUrl: './course-card.component.html',
   styleUrls: ['./course-card.component.scss'],
 })
-export class CourseCardComponent {}
+export class CourseCardComponent implements OnInit {
+  @Input() course!: Course;
+  @Input() isEditable = true;
+  authorsNames: string[] = [];
+
+  ngOnInit() {
+    this.authorsNames = mapAuthorIdsToNames(
+      this.course.authors,
+      mockedAuthorsList
+    );
+  }
+
+  @Output() showCourse = new EventEmitter<Course>();
```

Comment:
You should move this declaration up, next to the other class variables.

Reply:
done!